beginner_source/onnx/intro_onnx.py

```diff
@@ -7,7 +7,8 @@
 ====================
 
 Authors:
-`Thiago Crepaldi <https://github.com/thiagocrepaldi>`_,
+`Ti-Tai Wang <https://github.com/titaiwangms>`_, `Justin Chu <https://github.com/justinchuby>`_,
```

Comment:
Is there any particular reason to remove the original author?

Reply:
He does not work in our team anymore.